src/supervisor/watchers/handlers/index.ts

```diff
@@ -91,9 +93,67 @@ const workloadWatchMetadata: Readonly<IWorkloadWatchMetadata> = {
     },
     listFactory: (namespace) => () => k8sApi.appsClient.listNamespacedStatefulSet(namespace),
   },
+  [WorkloadKind.DeploymentConfig]: {
+    /** https://docs.openshift.com/container-platform/4.7/rest_api/workloads_apis/deploymentconfig-apps-openshift-io-v1.html */
+    endpoint: '/apis/apps.openshift.io/v1/watch/namespaces/{namespace}/deploymentconfigs',
+    handlers: {
+      [DELETE]: deploymentConfigWatchHandler,
+    },
+    listFactory: (namespace) => () => k8sApi.customObjectsClient.listNamespacedCustomObject("apps.openshift.io", "v1", namespace, "deploymentconfigs"),
+  },
 };
 
-export function setupInformer(namespace: string, workloadKind: WorkloadKind) {
+async function isSupportedWorkload(
+  namespace: string,
+  workloadKind: WorkloadKind,
+): Promise<boolean> {
+  if (workloadKind !== WorkloadKind.DeploymentConfig) {
+    return true;
+  }
+
+  try {
+    const pretty = undefined;
+    const continueToken = undefined;
+    const fieldSelector = undefined;
+    const labelSelector = undefined;
+    const limit = 1; // Try to grab only a single object
+    const attemptedApiCall = await k8sApi.customObjectsClient.listNamespacedCustomObject(
+      'apps.openshift.io',
+      'v1',
+      namespace,
+      'deploymentconfigs',
+      pretty,
+      continueToken,
+      fieldSelector,
+      labelSelector,
+      limit,
+    );
+    return (
+      attemptedApiCall !== undefined &&
+      attemptedApiCall.response !== undefined &&
+      attemptedApiCall.response.statusCode !== undefined &&
+      attemptedApiCall.response.statusCode > 200 &&
+      attemptedApiCall.response.statusCode < 300
```

Comment:
```suggestion
      Boolean(attemptedApiCall?.response?.statusCode > 200 && attemptedApiCall?.response?.statusCode > 200)
```

Reply:
`Object is possibly 'undefined'.ts(2532)` ;)

Reply:
oh, missed the first statement